source/core/ScriptDomain.cpp

```diff
@@ -96,69 +99,74 @@ namespace GTA
 			Log("[ERROR]", "Caught fatal unhandled exception:", Environment::NewLine, args->ExceptionObject->ToString());
 		}
 	}
-
-	ScriptDomain::ScriptDomain() : _appdomain(System::AppDomain::CurrentDomain), _executingThreadId(Thread::CurrentThread->ManagedThreadId), _runningScripts(gcnew List<Script ^>()), _taskQueue(gcnew Queue<IScriptTask ^>()), _pinnedStrings(gcnew List<IntPtr>()), _scriptTypes(gcnew List<Tuple<String ^, Type ^> ^>()), _recordKeyboardEvents(true), _keyboardState(gcnew array<bool>(256))
+	ScriptDomain::ScriptDomain() :
+		_appDomain(System::AppDomain::CurrentDomain),
+		_executingThreadId(Thread::CurrentThread->ManagedThreadId)
 	{
 		sCurrentDomain = this;
 
-		_appdomain->AssemblyResolve += gcnew ResolveEventHandler(&HandleResolve);
-		_appdomain->UnhandledException += gcnew UnhandledExceptionEventHandler(&HandleUnhandledException);
+		_appDomain->AssemblyResolve += gcnew ResolveEventHandler(&HandleResolve);
```

Comment:
That's not how ScriptHookVDotNet works. Every ScriptDomain manages its very own AppDomain. Creating a new ScriptDomain instance also creates a new AppDomain (line 125, in ```ScriptDomain::Load```).